### PR TITLE
git-reset: add page

### PR DESCRIPTION
**Pull request by eliotsykes**

### CODE REVIEW

File: pages/common/git-reset.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# git reset
+
+> Undo commits. Reset current HEAD to the specified state.
+
+- Undo last commit, keep the changes:
+
+`git reset HEAD~1`
+
+- Undo last commit and permanently delete the changes:
```
Comment (waldyrious) [inline pages/common/git-reset.md]: Maybe mention that staged changes are also discarded.

File: pages/common/git-reset.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# git reset
+
+> Undo commits. Reset current HEAD to the specified state.
+
+- Undo last commit, keep the changes:
```
Comment (waldyrious) [inline pages/common/git-reset.md]: Just to be clearer about where the changes are kept, I'd be explicit and say "keep the changes in the filesystem".

File: pages/common/git-reset.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# git reset
+
+> Undo commits. Reset current HEAD to the specified state.
```
Comment (waldyrious) [inline pages/common/git-reset.md]: Git reset has two common uses: undoing commits, and unstaging changes. Both should be mention in the description.

I'd suggest "Undo commits or unstage changes, by resetting the current git HEAD to the specified state."

**Comment by waldyrious** [comment]
Very nice and useful command :) please take a look at the inline comments.

I plan to make a PR adding the git-reset commands for unstaging changes, from notes I have collected elsewhere, so don't worry about adding those in :)

**Comment by eliotsykes** [comment]
Improvements made - really appreciate the feedback @waldyrious, thank you :-)

File: pages/common/git-reset.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# git reset
+
+> Undo commits or unstage changes, by resetting the current git HEAD to the specified state.
+
+- Undo last commit, keep the changes in the local filesystem:
```
Comment (waldyrious) [inline pages/common/git-reset.md]: Maybe be more specific and say "keep its changes"?

**Comment by waldyrious** [comment]
Happy to help, @eliotsykes :) I think the page is good to merge as-is. I'll propose further changes soon, to incorporate the unstaging behavior.

**Comment by agnivade** [comment]
:+1:

**Pull request merged**